main/tasks/power_management_task.c

```diff
@@ -143,20 +143,21 @@ void POWER_MANAGEMENT_task(void * pvParameters)
     }
 }
 
-// all values input below 60 result in 38 but increases exponentially such that an input of 70 results in 100
+// Fan speed between 33 min and 100 max based on input,
+// proportional to temp increase between 50 and THROTTLE_TEMP
```

Comment:
Does this comment make sense? Did you mean "proportional temp increase between 40 and THROTTLE_TEMP"

Reply:
I reworded the comment, hopefully it makes more sense